jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -255,6 +255,7 @@ with what features/services are supported.
         <registryKey key="aws.feature.ecsCloudDebug" description="Enables the Cloud Debug for ECS set of features" restartRequired="false" defaultValue="true"/>
         <registryKey key="aws.feature.ecsExec" description="Enables the features and actions associated with ECS Execute Command" restartRequired="false" defaultValue="false"/>
         <registryKey key="aws.feature.dynamoDb" description="Enables the features and actions associated with DynamoDB" restartRequired="false" defaultValue="true"/>
+        <registryKey key="aws.feature.moreResources.mutation" description="Enables the Create/ Update/ Delete features and actions associated with More resources" restartRequired="false" defaultValue="false"/>
```

Comment:
nit: spacing with the `/` in the description looks weird either do "create/update/delete" or "create / update / delete" is "More resources" a proper-noun? Should it just be "more resources"?

Reply:
Changed to create/update/delete